stickyuploads/tests/test_views.py

```diff
@@ -33,13 +33,14 @@ def test_upload(self):
             response = self.client.post(self.url, data)
         self.assertEqual(response.status_code, 200)
         result = json.loads(response.content.decode('utf-8'))
-        storage = TempFileSystemStorage()
         filename = os.path.basename(self.temp_name)
+        # We can't test stored value because there are unknowns
```

Comment:
Yes because `mkdtemp`  will generate random directory and we do not know it's name.